docs/micro-rdk/base/_index.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Micro-RDK Base Models"
+linkTitle: "Base"
+weight: 30
+type: "docs"
+description: "The micro-RDK support for moving platforms that the other parts of a mobile robot attach to."
+images: ["/icons/components/base.svg"]
+tags: ["base", "components", "micro-rdk"]
+no_list: true
+# SMEs: Nick M., Gautham V.
+---
+
+A base is the moving platform that the other parts of a mobile robot attach to.
+For more information and models supported by [the RDK (features provided by the full version of `viam-server`)](/internals/rdk/), see the [Base component](/components/base/).
```

Comment:
```suggestion
For more information and models supported by [the RDK (the full version of `viam-server`)](/internals/rdk/), see [Base Component](/components/base/).
```
[opt] Nice. I think the link text can be shorter on these. And then (nit) I think either capitalize to match the literal title of the doc, or don't capitalize "base" (/other component name)?

Reply:
I think using the glossary entry will make this a bit prettier.
```suggestion
For more information and models supported by the {{< glossary_tooltip term_id="rdk" text="rdk" >}}, see the [Base Component](/components/base/).
```